tests/Http/Controllers/Api/Entries/DeleteControllerMediaS3Test.php

```
@@ -0,0 +1,578 @@
+            $this->assertCount(0, $audios, 'Unexpected number of audio files remaining');
+            $this->assertCount(0, $videos, 'Unexpected number of video files remaining');
+
+            //now remove all the leftover fake files
+            Storage::disk('entry_original')->deleteDirectory($this->project->ref);
+            Storage::disk('entry_thumb')->deleteDirectory($this->project->ref);
+            Storage::disk('audio')->deleteDirectory($this->project->ref);
+            Storage::disk('video')->deleteDirectory($this->project->ref);
+        } catch (Throwable $e) {
+            $this->logTestError($e, $response);
+        }
+    }
```

PHPMD (2.15.0)

The method test_it_should_delete_all_media_and_leave_entries_in_place is not named in camelCase. (Controversial Rules)

(CamelCaseMethodName)